easy_my_coop_payment_term/models/subscription_request.py

```diff
@@ -0,0 +1,9 @@
+from odoo import api, fields, models
+
+class SubscriptionRequest(models.Model):
+    _inherit = "subscription.request"
+
+    def create_invoice(self, partner):
+        invoice = super().create_invoice(partner)
+        invoice.payment_term_id = self.env["res.company"]._company_default_get().default_payment_term
```

Comment:
What happens if default_payment_term is undefined?

Reply:
Then we fall back on the current behavior : payment term is undefined in the invoice. I think this quiet behavior is preferable to raising an error

Reply:
OK!